package.json

```diff
@@ -24,7 +24,7 @@
     "typescript": "4.4.3"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "HTTPS=true craco start",
```

Comment:
Thanks, that's a good idea to use `https` in local, could you create a new script `start:https`, and add the usage in README.md?

BTW, we are going to refactor this repo with `create-liff-app`.
https://github.com/line/liff-playground/pull/8
I will keep this script and migrate it to `vite`.